src/main/java/frc/robot/RobotContainer.java

```diff
@@ -95,29 +98,32 @@
 
   // ---------------------------------------Commands--------------------------------------------
   private final AutonDriveCommand autonDriveCommand = new AutonDriveCommand(falconDriveSubsystem, limelight);
+  private final PathWeaverCommand pathWeaverCommand = new PathWeaverCommand(falconDriveSubsystem);
 
   private final DriveCommand driveCommand = new DriveCommand(falconDriveSubsystem, mainController, driveConditioner);
   // Starts shooting and turns on the limelight.
-  private final Command shootCommand = new RunCommand(() -> shooterSubsystem.shoot(MoPrefs.getShooterHoodSetpoint()),
-      shooterSubsystem).alongWith(new RunCommand(limelight::lightsOn, limelight));
+  private final Command shootCommand = new RunCommand(
+      () -> shooterSubsystem.shoot(MoPrefs.get(MoPrefsKey.SHOOTER_HOOD_SETPOINT)), shooterSubsystem)
+          .alongWith(new RunCommand(limelight::lightsOn, limelight));
   // Deploys the shooter hood, starts shooting, and runs the intake.
   // 10 seconds later, all of those stop, and then drive off of the initiation
   // line.
   private final Command shootFromLine = new ParallelCommandGroup(
-      new RunCommand(() -> shooterSubsystem.shoot(MoPrefs.getShooterHoodSetpoint()), shooterSubsystem).withTimeout(10),
+      new RunCommand(() -> shooterSubsystem.shoot(MoPrefs.get(MoPrefsKey.SHOOTER_HOOD_SETPOINT)), shooterSubsystem)
+          .withTimeout(10),
       new RunCommand(intakeSubsystem::runIntakeFwd, intakeSubsystem).withTimeout(10))
           .andThen(new RunCommand(() -> falconDriveSubsystem.drive(0.5, 0), falconDriveSubsystem));
   // Drives forward to the wall, then starts shooting at a different angle than
   // normal(one tailored for shooting up against the wall)
   private final Command shootFromWall = new RunCommand(() -> falconDriveSubsystem.drive(0.5, 0)).withTimeout(5).andThen(
-      new ParallelCommandGroup(
-          new RunCommand(() -> shooterSubsystem.shoot(MoPrefs.getShootFromWallHoodSetpoint()), shooterSubsystem)),
+      new ParallelCommandGroup(new RunCommand(
+          () -> shooterSubsystem.shoot(MoPrefs.get(MoPrefsKey.SHOOT_FROM_WALL_HOOD_SETPOINT)), shooterSubsystem)),
```

Comment:
This looks like it was branched off of #26.
If that's the case, then we'll probably want to avoid merging it until #26 is closed, since this is going to have all the same changes.